angular-chart.js

```diff
@@ -267,6 +272,23 @@
       return [r, g, b];
     }
 
+    function rgbStringToRgb (color) {
```

Comment:
This can be greatly simplified by using regexp and the `Number` javascript core function which parses a string into a Number, e.g.

```
function rgbStringToRgb (color) { 
  var match = color.match(/^rgba?\(([\d,\.]+)\)$/);
  if (! match) throw new Error('Cannot parse rgb value');
  color = match[1].split(','); 
  return color.map(Number);
}
```